### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#854] [General]
I Have Website With Extensions. (https://scratchextensions.7m.pl/) and im asking can my website will be in trusted sources list?
also can i be extension maker

**Comment by PURPL3-CAT** [comment]
If you want to get your extensions unsandboxed, check [CONTRIBUTING.MD](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md). I don't think It's likely there will be an exception for your page in specific.

**Comment by David-Orangemoon** [comment]
> I Have Website With Extensions. (https://scratchextensions.7m.pl/) and im asking can my website will be in trusted sources list? also can i be extension maker

You could pull the extensions from your site to the turbowarp repository

**Comment by GarboMuffin** [comment]
> can my website will be in trusted sources list

no, sorry

> can i be extension maker

you are welcome to submit extensions to this repository
